internal/component/common/kubernetes/event.go

```diff
@@ -22,10 +22,10 @@ const (
 
 type queuedEventHandler struct {
 	log   log.Logger
-	queue workqueue.RateLimitingInterface
+	queue workqueue.TypedRateLimitingInterface[Event]
```

Comment:
I suppose `RateLimitingInterface` is deprecated?

Reply:
Correct